OracleLinuxDevelopers/oraclelinux8/golang/latest/Dockerfile

```diff
@@ -1,10 +1,10 @@
 # Copyright (c) 2020 Oracle and/or its affiliates.
 # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
-FROM oraclelinux:8-slim
+FROM ghcr.io/djelibeybi/oraclelinux8-compat:8-slim
```

Comment:
To end users this 'strange' repo would looks sus.  Can you add some comments to each Dockerfile to explain what it is, and what the plan is?

Reply:
Whoops, that was for testing purposes. I need to merge the compat image into this repo and change the URL. It will be `ghcr.io/oracle/oraclelinux8-compat:8-slim` eventually.